### ISSUE TITLE
"Datenpunkt Eigenschaften" Berechnung, Fehler im Diagramm

### ISSUE THREAD

**Issue by MubiTec** [Zefau/ioBroker.jarvis#2095]
*Please note that you may restore an old layout via jarvis adapter configuration directly within ioBroker

**Prerequisites / Voraussetzungen**
- [ x] Which jarvis version are you running / Welche Version von jarvis nutzt du? 3.1.0
- [ ] Fehler / Screenshot der Browser Konsole (F12), siehe [Fehleranalyse und -Meldung](https://github.com/Zefau/ioBroker.jarvis/wiki/de-Errors)

**Describe the bug / Beschreibe den Fehler**
Ich habe eine Berechnung in den "Datenpunkt Eigenschaften", ich teile dort den Wert durch 10.
Zeige ich diesen Datenpunkt dann im Historygraph, in dem ich "use device value function" aktiviere, schlägt das Faktor 10 geringere Ergebnis als Linie durch. Klicke ich jedoch auf einen Datenpunkt im Chart, so wird nur der Wert "-" angezeigt. Deaktiviere ich "use device value function" wieder, ist natürlich der alte Wert da, aber ich kann diesen nun auch in den Werten sehen.
Beispiele für die Reihe Ping in den Bildern
![Screenshot_20221209-231614_Chrome](https://user-images.githubusercontent.com/44448393/206804595-85b6f9da-7897-4d75-a346-4058e6e50e07.jpg)
![Screenshot_20221209-231534_Chrome](https://user-images.githubusercontent.com/44448393/206804599-a33d655d-f3ce-4737-a596-6b54660de4b0.jpg)

**Comment by mcuiobroker** [comment]
Wo kann man das aktivieren ?
`use device value function`

**Comment by MubiTec** [comment]
Im widget, ganz rechts unten im Bild
![Screenshot_20221209-232744_Chrome](https://user-images.githubusercontent.com/44448393/206805865-d0e2f0cd-5a3a-4342-9f2b-eefcac70fd44.jpg)

**Comment by Zefau** [comment]
Bei mir funktioniert es. Kannst du mal - wenn du die Einstellung aktiviert hast - die Seite komplett neu laden. Geht es dann?

**Comment by MubiTec** [comment]
Nee, leider nicht besser. Habs nun auch noch mal im frisch heruntergeladenen Firefox probiere.

Sobald die Einstellung wieder abgewählt ist, funktioniert es wieder 

![Uploading Screenshot_20230121_210046_Chrome.jpg…]()

**Comment by MubiTec** [comment]
Nee, leider nicht besser. Habs nun auch noch mal im frisch heruntergeladenen Firefox probiere.

Sobald die Einstellung wieder abgewählt ist, funktioniert es wieder 

![Uploading Screenshot_20230121_210046_Chrome.jpg…]()

**Comment by MubiTec** [comment]
![Screenshot_20230121_210046_Chrome](https://user-images.githubusercontent.com/44448393/213885241-d89ea45b-4eb2-43ac-a162-92ce829e67cd.jpg)
![Screenshot_20230121_210025_Chrome](https://user-images.githubusercontent.com/44448393/213885249-195eb605-7742-4708-b813-6786ca4a6140.jpg)
![Screenshot_20230121_205943_Chrome](https://user-images.githubusercontent.com/44448393/213885255-01a451c5-74ef-46ca-a50e-c77b6c18e643.jpg)

**Comment by Zefau** [comment]
Zeig mal bitte noch die Konfiguration des Geräts mit dem Datenpunkt (Einheit und Formel)
